client/src/components/overview/ProductInfo.jsx

```diff
@@ -1,33 +1,34 @@
 import React, { useState } from 'react';
+import calcStarImg from '../shared/calcStarImg.jsx';
 
-const ProductInfo = function ({ product, reviewsMeta }) {
+const ProductInfo = function ({ product, reviewMetaData }) {
   let sum = 0;
   let reviews = 0;
-  for (const key in reviewsMeta.ratings) {
-    sum += Number(reviewsMeta.ratings[key]) * Number(key);
-    reviews += Number(reviewsMeta.ratings[key]);
+  for (const key in reviewMetaData.ratings) {
```

Comment:
You could use the function I wrote in the shared folder to get the average and total number of reviews if you want:

https://github.com/Audacious-Alder/FEC/blob/main/client/src/components/shared/calcAvgTotalReviews.js

Reply:
Thanks for the suggestion, I just fixed it!